android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -786,21 +806,4 @@ public void run() {
         });
     }
 
-    public static void getDaysBetweenDates(Date startdate, Date enddate) {
```

Comment:
it was done by android studio. according to standards, static methods go on top.